doc/use-on-icub-head.md

```diff
@@ -0,0 +1,62 @@
+# Use the robotology-superbuild on the icub-head 
+
+## Introduction 
+One of the main goals of the robotology-superbuild is to simplify downloading and building all the software 
+that is necessary to have on the main control computer contained in the head of the iCub robot, whose hostname 
+is `icub-head` (or `pc104` in older iCub versions). This documentation provides the basic information necessary to 
+use the robotology-superbuild on the computer contained in the head of the iCub robot. 
+
+**Warning: if your iCub robot is not currently using the robotology-superbuild, do not attempt to migrate it
+without coordinating with the iCub support.  For any doubt, please open an issue at https://github.com/robotology/icub-support.**
+
+## Prepare your system 
+The operating system contained in the `icub-head`/`pc104` is tipically installed by IIT, for more information on it, please
+check the relevant documentation at http://wiki.icub.org/wiki/The_Linux_on_the_pc104 . On this machine, 
+in `/home/icub/.bashrc_iCub` a script containing several enviroment variables definitions is provided. If you want to  migrate to use
+the robotology-superbuild, you need to remove the existing `.bashrc_iCub` file, and substitute it with the one provided in https://git.robotology.eu/MBrunettini/icub-environment/raw/master/home/bashrc_iCub_superbuild .
+After you modified the `.bashrc_iCub` script, reboot the computer and  in a new shell check che values of the `ROBOTOLOGY_SUPERBUILD_SOURCE_DIR` and `YARP_ROBOT_NAME` env variables.
+If `ROBOTOLOGY_SUPERBUILD_SOURCE_DIR` contains `/usr/local/src/robot/robotology-superbuild`, and `YARP_ROBOT_NAME` contains the string specific to your robot,
+then the modification of the `.bashrc_iCub` was succesful. 
```

Comment:
typo: successful.

Reply:
Thanks, fixed  in https://github.com/robotology/robotology-superbuild/commit/8fd2aed342f55f02fe780a38a89995f4939145ac .  (For the future: you  can also directly suggest modifications using https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request , it is quite handy especially to accept them.